threads/client-tcp.c

```diff
@@ -0,0 +1,95 @@
+#include <stdlib.h>
+#include <stdio.h>
+#include <sys/types.h>
+#include <sys/socket.h>
+#include <netinet/in.h>
+#include <netdb.h>
+#include <unistd.h>
+#include <inttypes.h>
+#include <string.h>
+
+/****************************************
+        Author: Tim Wood
+        with a little help from
+        http://beej.us/guide/bgnet/
+****************************************/
+
+int main(int argc, char ** argv)
```

Comment:
Is this file identical to the one in sockets?  If so we could just delete it.  I guess it is fine to keep it for now and maybe later we will make this a multi-threaded client.